internal/cli/opsmanager/clusters/describe.go

```diff
@@ -40,20 +40,34 @@ func (opts *DescribeOpts) initStore() error {
 	return err
 }
 
-func (opts *DescribeOpts) Run() error {
-	result, err := opts.store.GetAutomationConfig(opts.ConfigProjectID())
+var describeTemplate = `ID	NAME	TYPE
+{{.ID}}	{{.ClusterName}}	{{.TypeName}}
+`
 
+func (opts *DescribeOpts) Run() error {
+	r, err := opts.cluster()
 	if err != nil {
 		return err
 	}
 
-	clusterConfigs := convert.FromAutomationConfig(result)
-	for _, rs := range clusterConfigs {
+	return output.Print(config.Default(), describeTemplate, r)
+}
+
+func (opts *DescribeOpts) cluster() (interface{}, error) {
+	if config.Output() == "" {
```

Comment:
should this be a `config.Simple` rather than empty string?

Reply:
simple will be "no output selected", and will be the default eventually, so if you want json you'll have to be explicit about it